migrations/acceptance-tests-v2.js

```diff
@@ -1,5 +1,7 @@
 var mongoose = require('mongoose');
-var _s = require('underscore.string');
+// This migration relied on "underscore.string": "^2.3.3"
+// It has been removed from package.json to lower the cost of upgrading dependencies
+var _s = undefined;//require('underscore.string');
```

Comment:
Better to keep the requirement and have the failure be explicit (“could not find `underscore.string` module” rather than “undefined is not an object”).